code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -147,7 +147,7 @@
 
 /obj/item/weapon/reagent_containers/pill/kelotane
 	name = "Kelotane pill"
-	desc = "Used to treat burns."
+	desc = "Used to treat burns. Contains 15 units of Kelotane."

```

Comment:
How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated.